src/i18n/en/en.json

```
@@ -836,7 +836,7 @@
   "pending": "pending",
   "All": "All",
   "View Sample": "View Sample",
-  "SYNC HSM": "SYNC HSM",
+  "Sync HSM": "Sync HSM",
```

Biome (2.5.6)

[error] The key Sync HSM was already declared.

(lint/suspicious/noDuplicateObjectKeys)